srcpkgs/v2ray/template

```diff
@@ -49,4 +50,7 @@ post_install() {
 	vmkdir etc/v2ray
 	vcopy release/config/*.json etc/v2ray/
 	vsv v2ray
+
+	# Void-specific documentation.
+	vdoc "${FILESDIR}/README.voidlinux"
```

Comment:
I feel like the readme is pointless if the service follows how every other service is configured.